code/api/src/migrations/20210325155429-5-userproducts.js.js

```diff
@@ -16,10 +16,10 @@ module.exports = {
         },
         allowNull: false
       },
-      crateId: {
+      productId: {
```

Comment:
@isabellevillasenor & @catherinemdean15 Just to confirm, these changes are ok, right?